DataTypes.mw.php

```diff
@@ -32,7 +32,7 @@
 	$args = func_get_args();
 	$key = array_shift( $args );
 	$language = array_shift( $args );
-	$message = new \Message( $key, $args );
```

Comment:
Better keep such changes separate from release stuff.

This code looks to be using the MediaWiki Message class, where the changed version uses the DataTypes one. (And yes, this message things is global cruft that should have been done via DI. More reason to move this PHP code into WB.git for now.)